client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -195,10 +195,10 @@ function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	const translate = useTranslate();
 
 	if ( hasPlanInCart && ! isJetpackNotAtomic ) {
-		return <span>{ translate( 'Unlimited email support' ) }</span>;
+		return <span>{ translate( 'Unlimited customer support via email' ) }</span>;
 	}
 
-	return <span>{ translate( 'Email support' ) }</span>;
+	return <span>{ translate( 'customer support via email' ) }</span>;
```

Comment:
Minor: I wonder if this should also be capitalised? Unless you lowercased it intentionally.

Reply:
@Patil2099: if you can make this update, I think this PR will be good to go. Thanks!